jx/scripts/ci-github.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+export GH_CREDS_PSW="$(jx step credential -s WHAT_TO_PUT_HERE)"
```

Comment:
@jstrachan - do you know what credentials you want to use for tests against github (as opposed to GHE) for the terraform tests?